packages/components/badge/src/BadgeItem.tsx

```diff
@@ -0,0 +1,51 @@
+import { forwardRef, HTMLAttributes, PropsWithRef } from 'react'
+
+import { styles, type StylesProps } from './BadgeItem.styles'
+import { DEFAULT_INTENT, DEFAULT_OVERFLOW_COUNT, DEFAULT_SIZE, DEFAULT_TYPE } from './config'
```

Comment:
IMO there is no need for a `config` file

Reply:
I think there is a need since there are some default values shared across different files.

Reply:
I think that we should come with an agreement regarding this because it's the first component to have a config file. I would try to avoid generic files like `config`, `utils` files as much as possible.

Reply:
Well, in fact it's a config exporting only constants... but, yep let's make an agreement on this